src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs

```diff
@@ -1051,26 +1050,9 @@ private static void OnSigChild(int reapAll)
         /// </summary>
         internal static void ConfigureTerminalForChildProcesses(int increment)
```

Comment:
I do not think you need the ConfigureTerminalForChildProcesses / ConfigureTerminalForChildProcessesInner split. ConfigureTerminalForChildProcesses can be the partial method.

Reply:
The `ConfigureTerminalForChildProcesses` method has a modifier so there can be `error CS8795: Partial method 'Process.ConfigureTerminalForChildProcesses(int)' must have an implementation part because it has accessibility modifiers.`
Removing the modifier leads to `src/System/Diagnostics/ProcessWaitState.Unix.cs(575,33): error CS0122: 'Process.ConfigureTerminalForChildProcesses(int)' is inaccessible due to its protection level`

Reply:
Hmmm, that's annoying.